lilypad/server/settings.py

```diff
@@ -35,12 +42,12 @@ def config(self) -> dict[str, Any]:
                 "client_url": "",
             },
             "local": {
-                "api_url": "http://localhost:8000/api",
-                "client_url": "http://localhost:8000",
+                "api_url": self.local_api_url,
```

Comment:
seems weird here to have `api_url` and `client_url` when it's really just `base_url` and then the inclusion of `/api`?

also curious where we're handling possible updates to the api version that's currently `v0` or if we just handle that in the client class?

Reply:
we will handle that in the client when we get to v1